ethcore/res/ethereum/classic.json

```diff
@@ -12,8 +12,8 @@
 				"registrar" : "0xc6d9d2cd449a754c494264e1809c50e34d64562b",
 				"homesteadTransition": "0x118c30",
 				"eip150Transition": "0x2625a0",
-				"eip155Transition": "0x7fffffffffffffff",
-				"eip160Transition": "0x7fffffffffffffff",
+				"eip155Transition": 3000000,
+				"eip160Transition": 3000000,
```

Comment:
Hm, is that bad? because I see all new changes to configuration are using decimal for blocks